src/store/session/actions.ts

```diff
@@ -42,6 +44,16 @@ export const actions: ActionTree<SessionState, RootState> = {
             rpc: rpcUrls,
           },
         },
+        'custom-ledger': ledgerProviderOptions,
+        'custom-trezor': {
+          ...trezorProviderOptions,
```

Comment:
If you are are trying to do a different deviation path you can specify it here. So it would be something like:

```ts
'custom-ledger': {
   ...ledgerProviderOptions,
   dPath: 'm/44'/0'/0'/0/0'
},
'custom-trezor: {
   ...trezorProviderOptions,
   dPath: 'm/44'/0'/0'/0/0'
}
```

Now, if you do this, especially in Ledger, you need to make sure the user is in the correct App on their device. So if they use the RSK app and try to connect to ETH they will get an error. 

I hope this answers your question.